### PR TITLE
add toggle to turn off links on user and group

### PR DESCRIPTION
**Pull request by jrwpatterson**
Signed-off-by: Joe Patterson <jrwpatterson@gmail.com>

## Hey, I just made a Pull Request!

adds a toggle off for links https://github.com/backstage/backstage/pull/14129#issuecomment-1323380210 as per this comment

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by jhaals** [comment]
Thanks for the pr @jrwpatterson! I took the liberty to tweak this PR by inverting the logic. This is primarly due to the fact that we already have a dedicated links card so I don't think we should render links by default in this card as well.

File: plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx [hunk 1]
```diff
@@ -191,7 +195,7 @@ export const GroupProfileCard = (props: { variant?: InfoCardVariants }) => {
                 secondary="Child Groups"
               />
             </ListItem>
-            <LinksGroup links={links} />
+            {props?.showLinks && <LinksGroup links={links} />}
```
Comment (tudi2d) [inline plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx:198]: looking at the types `props` should be defined here, right? it's just `showLinks` that might be `undefined`.

```suggestion
            {props.showLinks && <LinksGroup links={links} />}
```

File: plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx [hunk 2]
```diff
@@ -130,7 +134,7 @@ export const UserProfileCard = (props: { variant?: InfoCardVariants }) => {
               </ListItemText>
             </ListItem>
 
-            <LinksGroup links={links} />
+            {props?.showLinks && <LinksGroup links={links} />}
```
Comment (tudi2d) [inline plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx:137]: same as above
```suggestion
            {props.showLinks && <LinksGroup links={links} />}
```

**Review by tudi2d — APPROVED** [review]
reviewing because of #14819 - looks good, nitpick on the conditional rendering (not blocking though)

**Review by Rugvip — APPROVED** [review]

**Pull request merged**